nss-capricious-finches/nutshell
# Pzc storage

# Link to Ticket
Closes [35](https://github.com/nss-capricious-finches/nutshell/projects/1#card-9858387)
# Description of Proposed Changes

Session storage now takes  user object with unique username and stores that user's id to session storage. 

Session storage now retrieves the sessionUser key as a number. 

Steps to Test

Login with existing user. 
See session storage is saved as sessionUser:## (where ## is the user id)

Login with new user
See session storage is saved as sessionUser:## (where ## is the user id)

```
git fetch --all
git checkout pzc-storage-id
```

# Impacted Areas in Application
login
signup
friends
chat
articles

# Mentions @username
@rsbabcock @DanielBabcock
